paywall/src/__tests__/data-iframe/cacheHandler.test.js

```diff
@@ -147,6 +153,7 @@ describe('cacheHandler', () => {
       it('overwrites an existing transaction', async () => {
         expect.assertions(1)
 
+        // HERE IS THE FAIL - memory driver mutates its input
```

Comment:
Hum. Why would any component that uses the cache know about the type of caching?

Reply:
this comment was an in-progress note, I forgot to remove it.